### PR TITLE
Po-Sheng Lee Hw1.

### PR DESCRIPTION
**Pull request by lpstntw**

### CODE REVIEW

**Comment by christinayu56** [comment]
Evaluation HW-01
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |     x      |              |            |
| **Coding strategy**         |    x       |              |            |
| **Presentation**            |    x       |              |            |
| **Achievement, creativity** |    x       |              |            |
| **Ease of access**          |      x     |              |            |

Remarks:

* Elaborate on above, especially for "needs work." I thought your coding was very easy to follow and accessible.
* Some specific praise? I liked how put links relevant to your bio at the bottom.
* Something I learned? I wasn't too sure about how to input links into markdown but was able to visualize it clearly through your code.
* Specific constructive criticism? Nothing in particular (but maybe your bio could've included more personal information!) 
* Something I know and that you, my peer, might like to know because it is relevant to something you struggled with. I agree that embedding the photo is very difficult (I struggled with it too) but I learned that committing the image into github is an easy solution.

**Comment by marydeniserobertson** [comment]
Evaluation
----------------------------------------------------
| Topic                                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**                    |      x     |              |            |
| **Coding strategy**              |      x     |              |            |
| **Presentation**                    |      x     |              |            |
| **Achievement, creativity**  |      x     |              |            |
| **Ease of access**                |      x     |              |            |

Remarks:

Hi Po-Sheng, thank you for the introduction. I really liked your photo! Your code was very easy to follow. I thought your introduction was informative, and you used all the required elements of markdown. I also saw how you linked your photo, which was useful, seeing I did not include one in my intro. I don't have any criticism, I think you did a great job! 


* Elaborate on above, especially for "needs work."
* Some specific praise?
* Something I learned?
* Specific constructive criticism?
* Something I know and that you, my peer, might like to know because it is relevant to something you struggled with.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,18 @@
...
+1. Comparison of ***land finance*** among China, Taiwan and Hong Kong.
+2. The role of court in the process of democratic consolidation and democratic retreat.
+3. Law and Economic Development
+4. Governmental Debt in China
+
+A sometime photographer and a podcast addict. Here are some podcasts I find useful.
+
+[SupChina Sinica](https://supchina.com/category/podcast/)
+
+[China Power](https://chinapower.csis.org/podcasts/)
```
Comment (elaineywchen) [inline README.md:18]: Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |      x     |              |            |
| **Coding strategy**         |      x     |              |            |
| **Presentation**            |   x        |              |            |
| **Achievement, creativity** |    x      |              |            |
| **Ease of access**          |     x      |              |            |

Remarks:
* Your code is organized very clearly, and presented very clearly.
* There were some small mistakes that I made in my code (forgetting to put a space after #), so I learned what my mistakes through reading your code.
* I don't have much else to say about your code, but your research sounds very interesting!

**Comment by vivekr98** [comment]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |     x      |              |            |
| **Coding strategy**         |    x       |              |            |
| **Presentation**            |      x     |              |            |
| **Achievement, creativity** |     x      |              |            |
| **Ease of access**          |      x     |              |            |

Remarks:

* Your code is clear and easy to follow
* I really like the layout you chose, great picture!
* Good use of links and lists!

**Comment by cysakamoto** [comment]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |     x      |              |            |
| **Coding strategy**         |    x       |              |            |
| **Presentation**            |      x     |              |            |
| **Achievement, creativity** |   x        |              |            |
| **Ease of access**          |     x      |              |            |

Great job! Neat page, with nice use of the markdown elements.

**Pull request closed without merging**